packages/metro-file-map/src/flow-types.js

```diff
@@ -472,4 +472,6 @@ export type WorkerMetadata = $ReadOnly<{
   content?: ?Buffer,
 }>;
 
-export type WorkerSetupArgs = $ReadOnly<{}>;
+export interface WorkerSetupArgs {
+  __future__?: false;
```

Comment:
this is another one of those "doesn't translate to TS well" (empty object types), at the moment tossed in a field, but realistically there is probably a better type all together?